packages/checkout-ui-extensions/package.json

```diff
@@ -25,6 +28,6 @@
     "@remote-ui/core": "2.1.x"
   },
   "devDependencies": {
-    "@shopify/generate-docs": "^0.8.3"
+    "@shopify/generate-docs": "^0.8.10"
```

Comment:
I was getting the same [tsc not found](https://github.com/Shopify/ui-extensions/pull/752#pullrequestreview-1315783793) error when trying this in a local clone of `ui-extensions`.

This fixes it for me:

```diff
diff --git a/packages/checkout-ui-extensions/package.json b/packages/checkout-ui-extensions/package.json
index cdb63713..85cebf9d 100644
--- a/packages/checkout-ui-extensions/package.json
+++ b/packages/checkout-ui-extensions/package.json
@@ -28,6 +28,7 @@
     "@remote-ui/core": "2.1.x"
   },
   "devDependencies": {
-    "@shopify/generate-docs": "^0.8.10"
+    "@shopify/generate-docs": "^0.8.10",
+    "typescript": "^4.9.0"
   }
 }
```

However, I don't understand why this is necessary. The [generate-docs dependencies](https://github.com/Shopify/generate-docs/blob/main/packages/generate-docs/package.json#L42) already list `typescript`.

After that, I ran into another mysterious error after running `yarn docs:checkout`:

<details>
<summary>TypeError: fsStat.Settings is not a constructor</summary>

```
% yarn docs:checkout                                   
yarn run v1.19.2
$ yarn workspace @shopify/checkout-ui-extensions build-docs
$ sh ./docs/build-docs.sh
$ /Users/kumar/src/github.com/Shopify/ui-extensions/packages/checkout-ui-extensions/node_modules/.bin/tsc --project docs/tsconfig.docs.json --types react --moduleResolution node --target esNext --module CommonJS
$ /Users/kumar/src/github.com/Shopify/ui-extensions/packages/checkout-ui-extensions/node_modules/.bin/generate-docs --overridePath ./docs/typeOverride.json --input ./src --output ./docs/generated
TypeError: fsStat.Settings is not a constructor
    at new Reader (/Users/kumar/src/github.com/Shopify/ui-extensions/node_modules/fast-glob/out/readers/reader.js:9:32)
    at new ReaderAsync (/Users/kumar/src/github.com/Shopify/ui-extensions/node_modules/fast-glob/out/readers/async.js:8:9)
    at new ProviderAsync (/Users/kumar/src/github.com/Shopify/ui-extensions/node_modules/fast-glob/out/providers/async.js:8:24)
    at getWorks (/Users/kumar/src/github.com/Shopify/ui-extensions/node_modules/fast-glob/out/index.js:58:22)
    at FastGlob (/Users/kumar/src/github.com/Shopify/ui-extensions/node_modules/fast-glob/out/index.js:11:19)
    at /Users/kumar/src/github.com/Shopify/ui-extensions/node_modules/globby/index.js:133:52
    at Array.map (<anonymous>)
    at module.exports (/Users/kumar/src/github.com/Shopify/ui-extensions/node_modules/globby/index.js:133:40)
    at async Promise.all (index 0)
node:fs:590
  handleErrorFromBinding(ctx);
  ^

Error: ENOENT: no such file or directory, open
    at Object.openSync (node:fs:590:3)
    at Object.writeFileSync (node:fs:2202:35)
    at /Users/kumar/src/github.com/Shopify/ui-extensions/node_modules/@shopify/generate-docs/dist/generate-docs/src/assembler.js:158:12 {
  errno: -2,
  syscall: 'open',
  code: 'ENOENT'
}
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
** Failed to generate docs
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed.
Exit code: 1
Command: /opt/dev/sh/nvm/versions/node/v16.19.1/bin/node
Arguments: /Users/kumar/src/github.com/Shopify/ui-extensions/.yarn/releases/yarn-1.19.2.js build-docs
Directory: /Users/kumar/src/github.com/Shopify/ui-extensions/packages/checkout-ui-extensions
Output:

info Visit https://yarnpkg.com/en/docs/cli/workspace for documentation about this command.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

</details>

I can't reproduce the error in `checkout-web` on Spin. It comes from `globby` but the versions of this library are matching. The `fs.stat` versions are different but that doesn't seem like it should matter. Both versions [export](https://github.com/nodelib/nodelib/blob/%40nodelib/fs.stat%402.0.4/packages/fs/fs.stat/src/index.ts#L40) `Settings`.